source/style-guide/style/code-examples.txt

```diff
@@ -163,7 +174,7 @@ output:
 
 .. code-block:: sh
 
-   $ docker info Get http:///var/run/docker.sock/v1.20/info: dial unix
+   docker info Get http:///var/run/docker.sock/v1.20/info: dial unix
```

Comment:
Does this code block contain input and output? If so, doesn't this make it difficult to copy / run in the terminal?